static/js/containers/LearnerPage_test.js

```diff
@@ -1008,6 +1013,18 @@ describe("LearnerPage", function() {
       });
     });
 
+    it('should show grades, if the user has a staff role', () => {
```

Comment:
Can you also add a test for hiding the dialog, similar to the one for `DashboardPage`?